### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by jamie-lemon**

### CODE REVIEW

**Review by JorjMcKie — APPROVED** [review]

**Comment by jamie-lemon** [comment]
We should remove references to fitz everywhere I think. I found 23 matches across 7 files:

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/story-class.rst:
  108        :arg archive: an :ref:`Archive` from which to load resources for rendering. Currently supported resource types are images and text fonts. If omitted, the story will not try to look up any such data and may thus produce incomplete output.
  109        
  110:          .. note:: Instead of an actual archive, valid arguments for **creating** an :ref:`Archive` can also be provided -- in which case an archive will temporarily be constructed. So, instead of `story = fitz.Story(archive=fitz.Archive("myfolder"))`, one can also shorter write `story = fitz.Story(archive="myfolder")`.
  111  
  112     .. method:: Place(Rect where)
  ...
  257          :arg fn:
  258              A callable taking a floating point `parameter` and returning a
  259:             `fitz.Rect()`. If the rect is empty, we assume the story will
  260              not fit and do not call `self.Place()`.
  261  
  ...
  392  * `elpos.rect_num` (int) -- count of rectangles filled by the story so far.
  393  
  394: * `elpos.page_num` (int) -- page number; only present when using `fitz.Story.write*()` functions.
  395  
  396  .. include:: ../footer.rst

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/textpage.rst:
  221     2. :ref:`TextPage` and corresponding method :meth:`Page.get_text` are **available for all document types**. Only for PDF documents, methods :meth:`Document.get_page_images` / :meth:`Page.get_images` offer some overlapping functionality as far as image lists are concerned. But both lists **may or may not** contain the same items. Any differences are most probably caused by one of the following:
  222  
  223:    3. The image's "transformation matrix" is defined as the matrix, for which the expression `bbox / transform == fitz.Rect(0, 0, 1, 1)` is true, lookup details here: :ref:`ImageTransformation`.
  224  
  225  

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/document.rst:
 1618    .. attribute:: Permissions
 1619  
 1620:     Contains the permissions to access the document. This is an integer containing bool values in respective bit positions. For example, if *doc.Permissions & fitz.PDF_PERM_MODIFY > 0*, you may change the document. See :ref:`PermissionCodes` for details.
 1621  
 1622      :type: int

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/annot.rst:
   83        :arg int dpi: desired resolution in dots per inch. If not `null`, the matrix parameter is ignored.
   84  
   85:       :arg colorspace: a colorspace to be used for image creation. Default is *fitz.csRGB*.
   86        :type colorspace: :ref:`Colorspace`
   87  
   ..
  190        :arg string name: the new name.
  191  
  192:       .. caution:: If you set the name of a 'Stamp' annotation, then this will **not change** the rectangle, nor will the text be layouted in any way. If you choose a standard text from :ref:`StampIcons` (the **exact** name piece after `"STAMP_"`), you should receive the original layout. An **arbitrary text** will not be changed to upper case, but be written in font "Times-Bold" as is, horizontally centered in **one line** and be shortened to fit. To get your text fully displayed, its length using :data:`fontsize` 20 must not exceed 190 points. So please make sure that the following inequality is true: `fitz.get_text_length(text, fontname="tibo", fontsize=20) <= 190`.
  193  
  194     .. method:: SetRect(Rect rect)

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/matrix.rst:
   69        A "rect" must be any Rect object.
   70  
   71:       *fitz.Matrix(1, 1)* and *fitz.Matrix(fitz.Identity)* create modifiable versions of the :ref:`Identity` matrix, which looks like *[1, 0, 0, 1, 0, 0]*.
   72  
   73     .. method:: Norm()

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/shape.rst:
   53     .. method:: Shape(Page page)
   54  
   55:       Create a new drawing. During importing PyMuPDF, the *fitz.Page* object is being given the convenience method *new_shape()* to construct a *Shape* object. During instantiation, a check will be made whether we do have a PDF page. An exception is otherwise raised.
   56  
   57        :arg page: an existing page of a PDF document.
   ..
  414           >>> morph = (point, matrix)
  415           >>> # ... recalculate the shape rectangle like so:
  416:          >>> shape.rect = (shape.rect - fitz.Rect(point, point)) * ~matrix + fitz.Rect(point, point)
  417  
  418        :type: :ref:`Rect`
  ...
  454        pieces = len(cols)  # number of pieces to draw
  455        beta = 360. / pieces  # angle of each piece of pie
  456:       center = fitz.Point(...)  # center of the pie
  457:       p0 = fitz.Point(...)  # starting point
  458        for i in range(pieces):
  459            p0 = shape.DrawSector(center, p0, beta,
  ...
  471        shape = page.new_shape() # start a new shape
  472        beta = -360.0 / n  # our angle, drawn clockwise
  473:       center = fitz.Point(...)  # center of circle
  474:       p0 = fitz.Point(...)  # start here (1st edge)
  475        points = [p0]  # store polygon edges
  476        for i in range(n):  # calculate the edges
  ...
  540    Stroke and fill colors can be specified as tuples or list of of floats from 0 to 1. These sequences must have a length of 1 (GRAY), 3 (RGB) or 4 (CMYK). For GRAY colorspace, a single float instead of the unwieldy *(float,)* or *[float]* is also accepted. Accept (default) or use `null` to not use the parameter.
  541  
  542:   To simplify color specification, method *getColor()* in *fitz.utils* may be used to get predefined RGB color triples by name. It accepts a string as the name of the color and returns the corresponding triple. The method knows over 540 color names -- see section :ref:`ColorDatabase`.
  543  
  544    Please note that the term *color* usually means "stroke" color when used in conjunction with fill color.

/Users/jamielemon/Desktop/freelance/artifex/SDK/MuPDF.NET/docs/api/pixmap.rst:
  521  1. Directly create a pixmap with *Pixmap(filename)* or *Pixmap(byterray)*. The pixmap will then have properties as determined by the image.
  522  
  523: 2. Open such files with *fitz.open(...)*. The result will then appear as a document containing one single page. Creating a pixmap of this page offers all the options available in this context: apply a matrix, choose colorspace and alpha, confine the pixmap to a clip area, etc.
  524  
  525  **SVG images** are only supported via method 2 above, not directly as pixmaps. But remember: the result of this is a **raster image** as is always the case with pixmaps [#f1]_.
  ...
  547  .. note::
  548      * Not all image file types are supported (or at least common) on all OS platforms. E.g. PAM and the Portable Anymap formats are rare or even unknown on Windows.
  549:     * Especially pertaining to CMYK colorspaces, you can always convert a CMYK pixmap to an RGB pixmap with *rgb_pix = fitz.Pixmap(fitz.csRGB, cmyk_pix)* and then save that in the desired format.
  550      * As can be seen, MuPDF's image support range is different for input and output. Among those supported both ways, PNG and JPEG are probably the most popular.
  551      * We also recommend using "ppm" formats as input to tkinter's *PhotoImage* method like this: *tkimg = tkinter.PhotoImage(data=pix.tobytes("ppm"))* (also see the tutorial). This is **very** fast (**60 times** faster than PNG).

File: docs/api/vars.rst [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+    Predefined Python list of valid :ref:`Base-14-Fonts`.
+
+    :type: list
+
+.. py:data:: csRGB
+
+    Predefined RGB colorspace *Colorspace(Utils.CS_RGB)*.
+
+    :type: :ref:`Colorspace`
```
Comment (jamie-lemon) [inline docs/api/vars.rst:22]: Isn't it `ColorSpace` everywhere here? (i.e. upper camel cased?)

File: docs/api/vars.rst [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+
+.. py:data:: TEXT_ALIGN_JUSTIFY
+
+    3 -- align justify.
+
+.. _TextPreserve:
+
+Text Extraction Flags
+---------------------
+Option bits controlling the amount of data, that are parsed into a :ref:`TextPage` -- this class is mainly used only internally in PyMuPDF.
```
Comment (jamie-lemon) [inline docs/api/vars.rst:155]: PyMuPDF mentioned a couple of times from here ...

**Review by jamie-lemon — COMMENTED** [review]
I noticed vars.rst mentioned PyMuPDF & Python a few times but haven't left individual comments against that. Please check!

**Review by jamie-lemon — COMMENTED** [review]
mainly discovered lots of fullstops at end of the short method descriptions

File: docs/api/document.rst [hunk 3]
```diff
@@ -1623,7 +1627,7 @@ This class represents a document. It can be constructed from a file or from memo
 
   .. attribute:: MetaData
 
-    Contains the document's meta data as a dictionary or `null` (if *is_encrypted=True* and *needPass=True*). Keys are *format*, *encryption*, *title*, *author*, *subject*, *keywords*, *creator*, *producer*, *creationDate*, *modDate*, *trapped*. All item values are strings or `null`.
+    Contains the document's meta data as a dictionary or `null` (if *IsEncrypted=True* and *NeedPass=True*). Keys are *format*, *encryption*, *title*, *author*, *subject*, *keywords*, *creator*, *producer*, *creationDate*, *modDate*, *trapped*. All item values are strings or `null`.
```
[the remark this answers is not in the document]
Comment (greendreamer) [reply docs/api/document.rst:1627]: will fix right now

**Pull request merged**